scripts/readme.md

```diff
@@ -18,6 +18,18 @@ where $PHASE is test, discovery, alpha or beta
 
 Within those directories the definitions of individual fields or registers are in separate YAML files.
 
+This repo also contains files like
+
+    $ROOT/openregister/registry-data/data/$PHASE/$REGISTER/meta.yaml
+
+These contain additional metadata about the register. Each key corresponds to system entries stored in the register, for example "friendly-name".
```

Comment:
Did you mean `register-name` rather than `friendly-name`?

Reply:
oops, yes I did